pkg/server/wire.go

```diff
@@ -210,6 +211,7 @@ var wireBasicSet = wire.NewSet(
 	mssql.ProvideService,
 	store.ProvideEntityEventsService,
 	unified.ProvideUnifiedStorageClient,
+	unifiedsearch.ProvideBuilders,
```

Comment:
In enterprise, we will need an alternative implementation that knows how to fill in the usage stats